packages/preview/bamdone-aiaa/0.1.0/template/main.typ

```diff
@@ -0,0 +1,229 @@
+//***************************************************************
+// AIAA TYPST TEMPLATE
+// 
+// The author of this work hereby waives all claim of copyright
+// (economic and moral) in this work and immediately places it 
+// in the public domain; it may be used, distorted or 
+// in any manner whatsoever without further attribution or notice
+// to the creator. The author is not responsible for any liability 
+// from the usage or dissemination of this code.
+//
+// Author: Isaac Weintraub, Alexander Von Moll
+// Date: 06 NOV 2023
+// BAMDONE!
+//***************************************************************
+
+#import "@preview/bamdone-aiaa:0.1.0": aiaa
```

Comment:
There are multiple missing imports here, e.g. `nomenclature` and `dropcap`.

Reply:
I have added the other imports as suggested. Thank you.